docs/features.md

```diff
@@ -0,0 +1,45 @@
+---
+Order: 20
+Title: Features
+Description: Features of the Cake.Issues.EsLint addin.
+---
+The [Cake.Issues.EsLint addin] provides the following features.
+
+# Basic features
+
+* Reads issues reported by ESLint.
+* Provides URLs for all issues.
+* Support for custom URL resolving using the [EsLintAddRuleUrlResolver] alias.
+
+# Supported log file formats
+
+* [EsLintJsonFormat] alias for reading issues from log files created by [ESLint json formatter].
+
+# Supported comment formats
+
+|                                                                    | Comment format                 | Remarks                        |
+|--------------------------------------------------------------------|--------------------------------|--------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.PlainText` |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Markdown`  |                                |
+| <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IssueCommentFormat.Html`      |                                |
+
+# Supported IIssue properties
```

Comment:
[MD025](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document
) multiple top level headings in the same document.